src/main/java/seedu/contax/logic/commands/BatchCommand.java

```diff
@@ -84,8 +106,8 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(resultOutput.toString());
     }
 
-    private List<Index> matchInputStringToIndex(Model model, SearchType searchType, String userInput)
-            throws CommandException {
+    private List<Index> matchInputStringToIndex(Model model,
```

Comment:
Perhaps you could use a more descriptive name (or have javadocs) since this method now does a lot more things.

Reply:
thx will adda java doc